packages/plugins/hardhat.config.ts

```diff
@@ -22,17 +22,14 @@ const config: HardhatUserConfig = {
     },
   },
   networks: {
-    localhost: {
-      url: 'http://localhost:8545',
-    },
```

Comment:
Why remove this?

Reply:
`http://localhost:8545` is the default port for the localhost Hardhat network, so there's no need to explicitly define this